orchagent/orch.cpp

```diff
@@ -171,31 +185,50 @@ void Consumer::drain()
         m_orch->doTask(*this);
 }
 
-bool Orch::addExistingData(const string& tableName)
+size_t Orch::addExistingData(const string& tableName)
 {
-    Consumer* consumer = dynamic_cast<Consumer *>(getExecutor(tableName));
+    auto consumer = dynamic_cast<Consumer *>(getExecutor(tableName));
     if (consumer == NULL)
     {
         SWSS_LOG_ERROR("No consumer %s in Orch", tableName.c_str());
-        return false;
+        return 0;
     }
 
-    consumer->refillToSync();
-    return true;
+    return consumer->refillToSync();
 }
 
 // TODO: Table should be const
-bool Orch::addExistingData(Table *table)
+size_t Orch::addExistingData(Table *table)
 {
     string tableName = table->getTableName();
     Consumer* consumer = dynamic_cast<Consumer *>(getExecutor(tableName));
     if (consumer == NULL)
     {
         SWSS_LOG_ERROR("No consumer %s in Orch", tableName.c_str());
-        return false;
+        return 0;
+    }
+
+    return consumer->refillToSync(table);
+}
+
+bool Orch::bake()
+{
+    SWSS_LOG_ENTER();
+
+    for(auto &it : m_consumerMap)
+    {
+        string executorName = it.first;
+        auto executor = it.second;
+        auto consumer = dynamic_cast<Consumer *>(executor.get());
+        if (consumer == NULL)
+        {
+            continue;
+        }
+        
+        size_t refilled = consumer->refillToSync();
+        SWSS_LOG_NOTICE("Add warm input: %s, %zd", executorName.c_str(), refilled);
```

Comment:
I prefer NOTICE since it is important and one time thing. The total lines are about 35.

---
In reply to: [209397186](https://github.com/Azure/sonic-swss/pull/572#discussion_r209397186) [](ancestors = 209397186)